static/js/dashboard.js

```diff
@@ -85,1162 +93,1411 @@ $(document).mouseup(function (e) {
-        editPanelFlag =true;
-    } else {
-        $('#app-container').hide();
-    }
-    $('.dbSet-container').show();
-
-    $('.dbSet-name').html(dbName)
-    $('.dbSet-dbName').val(dbName)
-    $('.dbSet-dbDescr').val(dbDescr)
-    $('.dbSet-jsonModelData').val(JSON.stringify(JSON.unflatten({
+  if ($(".panelEditor-container").css("display") !== "none") {
+    $(".panelEditor-container").hide();
+    editPanelFlag = true;
+  } else {
+    $("#app-container").hide();
+  }
+  $(".dbSet-container").show();
+
+  $(".dbSet-name").html(dbName);
+  $(".dbSet-dbName").val(dbName);
+  $(".dbSet-dbDescr").val(dbDescr);
+  $(".dbSet-jsonModelData").val(
+    JSON.stringify(
+      JSON.unflatten({
         description: dbDescr,
         name: dbName,
         timeRange: timeRange,
         panels: localPanels,
         refresh: dbRefresh,
-    }), null, 2))
-    $('.dbSet-dbName').on("change keyup paste", function () {
-        dbName = $('.dbSet-dbName').val()
-        $('.dbSet-name').html(dbName)
-    })
-    $('.dbSet-dbDescr').on("change keyup paste", function () {
-        dbDescr = $('.dbSet-dbDescr').val()
-        $('.dbSet-dbDescr').html(dbDescr)
-        $('.dbSet-jsonModelData').val(JSON.stringify(JSON.unflatten({
-            description: dbDescr,
-            name: dbName,
-            timeRange: timeRange,
-            panels: localPanels,
-            refresh: dbRefresh,
-        }), null, 2))
-    })
-
-    //get dashboard data from database
-    $.ajax({
-        method: "get",
-        url: "api/dashboards/" + dbId,
-        headers: {
-            'Content-Type': 'application/json; charset=utf-8',
-            'Accept': '*/*'
-        },
-        dataType: 'json',
-        crossDomain: true,
-    }).then(function (res) {
-        console.log(JSON.stringify(res))
-        $('.dbSet-jsonModelData').val(JSON.stringify(JSON.unflatten(res), null, 2))
-    })
-
-    showGeneralDbSettings();
-    addDbSettingsEventListeners();
+      }),
+      null,
+      2
+    )
+  );
+  $(".dbSet-dbName").on("change keyup paste", function () {
+    dbName = $(".dbSet-dbName").val();
+    $(".dbSet-name").html(dbName);
+  });
+  $(".dbSet-dbDescr").on("change keyup paste", function () {
+    dbDescr = $(".dbSet-dbDescr").val();
+    $(".dbSet-dbDescr").html(dbDescr);
+    $(".dbSet-jsonModelData").val(
+      JSON.stringify(
+        JSON.unflatten({
+          description: dbDescr,
+          name: dbName,
+          timeRange: timeRange,
+          panels: localPanels,
+          refresh: dbRefresh,
+        }),
+        null,
+        2
+      )
+    );
+  });
+
+  //get dashboard data from database
+  $.ajax({
+    method: "get",
+    url: "api/dashboards/" + dbId,
+    headers: {
+      "Content-Type": "application/json; charset=utf-8",
+      Accept: "*/*",
+    },
+    dataType: "json",
+    crossDomain: true,
+  }).then(function (res) {
+  
+    console.log(JSON.stringify(res));
+    $(".dbSet-dbName").val(res.name);
```

Comment:
I am populating the dbName and dbDescription fields with the value extracted from the get API. The values will be displayed in the Dashboard UI